unit_tests/test_cluster.py

```diff
@@ -92,38 +92,58 @@ def test_search_system_log_specific_log(self):  # pylint: disable=invalid-name
     def test_search_system_interlace_reactor_stall(self):  # pylint: disable=invalid-name
         self.node.system_log = os.path.join(os.path.dirname(__file__), 'test_data', 'system_interlace_stall.log')
 
-        self.node._read_system_log_and_publish_events()
+        self.node._read_system_log_and_publish_events(start_from_beginning=True)
 
-        events_file = open(EVENTS_PROCESSES['MainDevice'].raw_events_filename, 'r')
-        events = []
-        for line in events_file.readlines():
+        with open(EVENTS_PROCESSES['MainDevice'].raw_events_filename, 'r') as events_file:
+            events = []
+            for line in events_file.readlines():
 
-            events.append(json.loads(line))
+                events.append(json.loads(line))
```

Comment:
The same thing done in another way later. Can you make it consistent? Also, you don't need call `fobj.readlines()` to read a file line-by-line. It's enough to use `for line in fobj`.

Reply:
Done